api/tests/functional-tests/backend/query/test_ops.py

```diff
@@ -1033,50 +1224,90 @@ def test_multiple_tables_in_args(
     ) in name_pairings
 
 
+def create_geospatial_inside_filter(
+    symbol_name: str,
+    value: Value,
+):
+    return Inside(
+        inside=Operands(
+            lhs=Symbol(
+                type=value.type, name=symbol_name, key="some_geo_attribute"
+            ),
+            rhs=value,
+        )
+    )
+
+
+def create_geospatial_outside_filter(
+    symbol_name: str,
+    value: Value,
+):
+    return Outside(
+        outside=Operands(
+            lhs=Symbol(
+                type=value.type, name=symbol_name, key="some_geo_attribute"
+            ),
+            rhs=value,
+        )
+    )
+
+
+def create_geospatial_intersects_filter(
+    symbol_name: str,
+    value: Value,
+):
+    return Intersects(
+        intersects=Operands(
+            lhs=Symbol(
+                type=value.type, name=symbol_name, key="some_geo_attribute"
+            ),
+            rhs=value,
+        )
+    )
+
+
 def _get_geospatial_names_from_filter(
     db: Session,
-    geodict: dict[
-        str,
-        list[list[list[list[float | int]]]]
-        | list[list[list[float | int]]]
-        | list[float | int]
-        | str,
-    ],
+    value: Value,
```

Comment:
nit: `value` is very generic; consider making this code easier to follow by using a more specific name

Reply:
See above. Resolving this as it is a duplicate.